frontend/pages/model/[modelId].tsx

```diff
@@ -62,13 +69,14 @@ export default function Model() {
 
   const error = MultipleErrorWrapper(`Unable to load model page`, {
     isModelError,
+    isCurrentUserError,
     isUiConfigError,
   })
   if (error) return error
 
   return (
     <Wrapper title={model ? model.name : 'Loading...'} page='marketplace' fullWidth>
-      {(isModelLoading || isUiConfigLoading) && <Loading />}
+      {isModelLoading && isCurrentUserLoading && isUiConfigLoading && <Loading />}
```

Comment:
This should be `{(isModelLoading || isCurrentUserLoading || isUiConfigLoading) && <Loading />}` because we want to show the loading component if any of them are loading.